### ISSUE TITLE
Unable to install version 1.2.1

### ISSUE THREAD

**Issue by mislav-eu** [loewexy/pdnsmanager#62]
Hi there.

I'm unable to install verison 1.2.1 - error in apache logs is:

> [Wed Nov 08 13:27:08.320136 2017] [:error] [pid 1197] [client XXX:57968] PHP Fatal error:  Uncaught Error: Call to a member function bind_param() on boolean in /var/www/html/pdnsmanager/api/install.php:126\nStack trace:\n#0 {main}\n  thrown in /var/www/html/pdnsmanager/api/install.php on line 126, referer: http://XXX/poweradmin/install.php

When I click "Install" button, nothing happens, so I went to check error logs and found this.

**Comment by mauricemeyer** [comment]
Do you have the `mysqli`-extension installed? It is needed. 

For Ubuntu, it is `phpX.X-mysql`

**Comment by mislav-eu** [comment]
Yes, it's installed. I've tried to download development version and I'm able to install that one. After installation is complete I can login without problems.

**Comment by mauricemeyer** [comment]
Okay, so please give me more information about your installation:

- Which OS are you using?
- Which version of PHP?
- Which PHP service: php-fpm, apache with mod_php?
- Please add the output of `php -i | grep mysql` and the output of a `phpinfo()` by php-fpm or mod_php.

**Comment by nbeernink** [comment]
I ran into same issue here. 

CentOS 7
I tried both php5.6 and 7.1 from remi repositories
PHP runs as mod_php via apache 2.4.
MariaDB 10.2

This really seems like a coding issue though. 

It seems https://github.com/loewexy/pdnsmanager/blob/d9f113c95dbe4d9220620fdc21e894bf25789b27/api/install.php#L27-L28 did not work properly, no additional tables were created but no error was shown. After running those queries manually I did get some errors;

```
MariaDB [pdns]> ALTER TABLE records
    ->   ADD CONSTRAINT records_ibfk_1 FOREIGN KEY (domain_id) REFERENCES domains (id) ON DELETE CASCADE;
ERROR 1005 (HY000): Can't create table `pdns`.`#sql-19fdf_96` (errno: 121 "Duplicate key on write or update")

MariaDB [pdns]> CREATE TABLE domainmetadata (
    ->     id INT AUTO_INCREMENT,
    ->     domain_id INT NOT NULL,
    ->     kind VARCHAR(32),
    ->     content TEXT,
    ->     PRIMARY KEY (id)
    -> ) Engine=InnoDB
ERROR 1050 (42S01): Table 'domainmetadata' already exists
```

The line it's complaining https://github.com/loewexy/pdnsmanager/blob/d9f113c95dbe4d9220620fdc21e894bf25789b27/api/install.php#L125-L126 is about inserting a user while the rest of the db tables do not exist yet.

The bleeding edge code seems to be fixed in the current code and I think the devs just need to do another release containing a fix for this. :)
